src/library_html5.js

```diff
@@ -2493,6 +2508,15 @@ var LibraryHTML5 = {
     targetCanvas = targetCanvas ? UTF8ToString(targetCanvas) : '';
     _emscripten_set_offscreencanvas_size_on_target_thread_js(targetThread, targetCanvas, width, height);
   },
+#else
+  emscripten_set_offscreencanvas_size__sig: 'iiii',
```

Comment:
This line is duplicated.. maybe lift it outside the `#if/#else`?

Reply:
f242a3e0fc87c29d66ccab0a0caa28871c631135